### ISSUE TITLE
No Dark Matter option

### ISSUE THREAD

**Issue by rabidhadron** [MarcBoule/Geodesics#5]
I'm using VCV Rack 0.6.2b with the 0.6.2 version of the Geodesics modules and there is no Dark Matter option under "Panel Theme".

**Comment by MarcBoule** [comment]
Correct, Dark Matter was introduced in the 0.6.3 release, which is currently in the build pipeline for the next update in the plugin manager (it should be out in a few days I believe, when Andrew does the next build). As it is now, unfortunately, you can either wait for the update to go through, or if you know how to compile plugins, you can get it from this github. Cheers!

**Comment by rabidhadron** [comment]
Thanks Marc for the super quick response. I'm happy to wait no problem.
